hack/tests/integration.sh

```diff
@@ -2,15 +2,34 @@
 
 set -eu
 
+source hack/lib/common.sh
+source hack/lib/test_lib.sh
 source hack/lib/image_lib.sh
 
-export OSDK_INTEGRATION_IMAGE="quay.io/example/memcached-operator:latest"
+TMPDIR="$(mktemp -d memcached-XXXX)"
```

Comment:
requires to be XXXX? Could not be a name related to the test?

Reply:
`XXXX` is a template string which will be substituted with random characters. See `mktemp -h` for more info.